mohawk/tests.py

```diff
@@ -689,20 +716,6 @@ def test_bewit_with_ext(self):
         expected = '123456\\1356420707\\kscxwNR2tJpP1T1zDLNPbB5UiKIU9tOSJXTUdG7X9h8=\\xandyandz'
         eq_(b64decode(bewit).decode('ascii'), expected)
 
-    def test_bewit_with_ext_and_backslashes(self):
```

Comment:
Well, it didn't, but since early failures are good I added stricter checks, so now they do. I split this test in two since it's technically two different issues, validation of the `id` and of the `ext`.